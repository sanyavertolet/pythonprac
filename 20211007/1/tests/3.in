(17, 33), (53, 6), (48, 9), (6, 38), (30, 49), (52, 30)
