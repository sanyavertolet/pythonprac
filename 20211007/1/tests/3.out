((53, 6), (30, 49), (52, 30))
